test/services/LibraryService.test.ts

```diff
@@ -13,17 +20,32 @@ const LIBRARY_WITH_URL: fhir4.Library = {
   url: 'http://example.com'
 };
 
-const LIBRARY: fhir4.Library = {
+const LIBRARY_WITH_NO_DEPS: fhir4.Library = {
   resourceType: 'Library',
+  id: 'testLibraryWithNoDeps',
+  status: 'draft',
+  url: 'http://example.com/testLibrary',
+  type: { coding: [{ code: 'logic-library' }] }
+};
+
+const LIBRARY_WITH_DEPS: fhir4.Library = {
+  resourceType: 'Library',
+  id: 'testLibraryWithDeps',
+  status: 'draft',
+  url: 'http://example.com/testLibraryWithDeps',
   type: { coding: [{ code: 'logic-library' }] },
-  id: 'test',
-  status: 'active'
+  relatedArtifact: [
```

Comment:
It may be useful to test one more level of traversal. Like A which depends on B which depends on C and making sure all 3 are included

Reply:
Good suggestion!